mobify/platform-scaffold
# More Merlin's command cleanups

This PR cleans up the Merlin's command code further, subsequent to #646. 

 **JIRA**: Preliminary for https://mobify.atlassian.net/browse/WEBDATA-69
 **Linked PRs**: (links to corresponding PRs, optional)

## Changes
- Use the `getCartBaseUrl` selector for all cart API URLs
- Clean up the promo code commands
- Use a single total fetch action for both tax estimates and post-promo-code estimates.

## How to test-drive this PR
- Preview Merlin's
- Make sure the cart works, including the shipping/tax estimate form
- Add the promo code `MAGIC` and see that the totals update
- Remove the promo code and see that they update again.
